test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json

```diff
@@ -2652,7 +2659,8 @@
                         "size": 385472253,
                         "size_": 385472253,
                         "file_format": "fastq.gz",
-                        "count": 2
+                        "count": 2,
```

Comment:
Should we map this field as None so it is not translated? I don't think it can ever be None.